trueline.sh

```diff
@@ -116,6 +116,24 @@ _trueline_venv_segment() {
     fi
 }
 
+_trueline_has_conda_env() {
+    printf "%s" "${CONDA_DEFAULT_ENV}"
+}
+
+_trueline_conda_env_segment() {
+    local conda_env="$(_trueline_has_conda_env)"
+    if [[ -n "$conda_env" ]]; then
+        local fg_color="$1"
+        local bg_color="$2"
+        local font_style="$3"
+        local segment="$(_trueline_separator)"
+        segment+="$(_trueline_content "$fg_color" "$bg_color" "$font_style" " ${TRUELINE_SYMBOLS[venv]} $conda_env")"
+        PS1+="$segment"
+        _last_color=$bg_color
+    fi
+}
+
```

Comment:
Remove this line and we are ready to go.

Reply:
removed this line took, stylized exactly the same as venv